images/prow-tests/Dockerfile

```diff
@@ -206,14 +210,10 @@ RUN go install github.com/gogo/protobuf/protoc-gen-gogofaster@${PROTOC_GEN_GOGOF
 RUN go install github.com/GoogleCloudPlatform/docker-credential-gcr@${DOCKER_CREDENTIAL_GCR_VERSION}
 
 # We do this instead of `go get` so the tools are locked to the Dockerfile's commit
-COPY . /go/src/knative.dev/test-infra
+COPY . /go/src/knative.dev/infra
 
 # Build custom tools in the container
-RUN cd /go/src/knative.dev/test-infra && go install ./tools/kntest/cmd/kntest
```

Comment:
these are just being removed?

we should probably pull in Chris' changes https://github.com/knative/infra/pull/73/files#diff-3a809bc2ddb42dd8cd4627d729d61452cc1ed660a8d82ae02476cf740e7740faR208-R212

Reply:
They are being installed but from the toolbox repo.

Reply:
Are you bringing in the whole toolbox? It seems @cardil is only bringing in whats needed see https://github.com/knative/infra/pull/79/files#diff-3a809bc2ddb42dd8cd4627d729d61452cc1ed660a8d82ae02476cf740e7740faR208-R212